Sources/Apodini/Configurations/Configuration.swift

```diff
@@ -22,6 +32,19 @@ public protocol ConfigurationCollection {
     @ConfigurationBuilder var configuration: Configuration { get }
 }
 
+extension Configuration {
+    // swiftlint:disable identifier_name
+    /// *For internal use only:* An array of the `command` of the configuration.
+    /// Used to allow iteration over the commands of a `ConfigurationBuilder`.
+    public var _commands: [ParsableCommand.Type] {
```

Comment:
@PSchmiedmayer As far as I tested, this needs to stay in the Configuration protocol  and public, otherwise the commands won't get added correctly to the cli, but just the default `EmptyCommand`